client/src/components/MultiJurisdictionAudit/Setup/Review/index.tsx

```diff
@@ -9,28 +9,38 @@ import { ISidebarMenuItem } from '../../../Atoms/Sidebar'
 import H2Title from '../../../Atoms/H2Title'
 import useAuditSettings from '../../useAuditSettings'
 import useContests from '../../useContests'
-import { IContest, ISampleSizeOption } from '../../../../types'
+import {
+  IContest,
+  ISampleSizeOptions,
+  ISampleSizeOption,
+} from '../../../../types'
 import useJurisdictions, { FileProcessingStatus } from '../../useJurisdictions'
 import { api, checkAndToast, testNumber } from '../../../utilities'
 import FormSection, {
   FormSectionDescription,
-  FormSectionLabel,
 } from '../../../Atoms/Form/FormSection'
 import ContestsTable from './ContestsTable'
 import SettingsTable from './SettingsTable'
 import { isSetupComplete } from '../../StatusBox'
 import useJurisdictionFile from '../Participants/useJurisdictionFile'
 import ConfirmLaunch from './ConfirmLaunch'
 import FormField from '../../../Atoms/Form/FormField'
+import ElevatedCard from '../../../Atoms/SpacedCard'
 
 const percentFormatter = new Intl.NumberFormat(undefined, {
   style: 'percent',
 })
 
-interface IStringSampleSize {
-  size: string
-  type: string | null
-  prob: number | null
+interface IStringSampleSizes {
+  [key: string]: {
+    size: string
```

Comment:
why use strings here? seems like numbers would be easier, since that's the API uses

Reply:
@jonahkagan Strings are what the form converts everything to, so it has a separate type for that so it's easier to track the conversion to strings going into the form and then remembering to convert back to numbers before sending it to the api.